test-fixtures/update_bitrise_steps.py

```diff
@@ -10,8 +10,17 @@
 # GitHub Access Token
 GITHUB_TOKEN = os.getenv("GITHUB_TOKEN")
 
+# Step versions that should NOT be updated
+LOCKED_VERSIONS = {
```

Comment:
I would add a comment why this is locked to this version:
The newest git-clone step changes the way the clone works breaks the decission to run UI test step due to the change in the variables available.... something like that..